.htaccess

```
@@ -67,6 +67,7 @@
   RewriteRule ^\.well-known/nodeinfo /public.php?service=nodeinfo [QSA,L]
   RewriteRule ^\.well-known/carddav /remote.php/dav/ [R=301,L]
   RewriteRule ^\.well-known/caldav /remote.php/dav/ [R=301,L]
+  RewriteRule ^\.well-known/change-password /settings/user/security [R=301]
```

```suggestion
  RewriteRule ^\.well-known/change-password /index.php/settings/user/security [R=301]
```

Otherwise this might fail if htaccess.RewriteBase is not configured.

---

Is there a reason you don't apply the "L" flag (last rule), as the other rewrites do?  Probably should.

---

@brad2014 I didn't add the L flag because I was not sure whether this actually was the last rule or not. But I guess, with the modifications of @juliushaertl, it definitely is, so I'll add that one